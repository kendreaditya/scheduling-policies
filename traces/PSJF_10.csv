1,1,20
2,4,19
3,6,18
4,8,17
5,10,16
6,12,15
7,14,14
8,16,13
9,18,12
10,20,11
11,22,10
12,24,9
13,26,8
14,28,7
15,30,6
16,32,5
17,34,4
18,36,3
19,38,2
20,40,1
