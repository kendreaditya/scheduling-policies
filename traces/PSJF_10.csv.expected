1,211
2,194
3,177
4,161
5,146
6,132
7,119
8,107
9,96
10,86
11,77
12,69
13,62
14,56
15,51
16,47
17,44
18,42
19,40
20,41
